src/components/editor/use-host-bridge.js

```diff
@@ -26,6 +22,12 @@ export function useHostBridge( post, editorRef ) {
 		[ editEntityRecord, post.id, post.type ]
 	);
 
+	const postTitleRef = useRef( post.title.raw );
+	const postContentRef = useRef( null );
+	if ( postContentRef.current === null ) {
+		postContentRef.current = serialize( parse( post.content.raw || '' ) );
+	}
```

Comment:
Explicitly [selectively instantiated](https://react.dev/reference/react/useRef#avoiding-recreating-the-ref-contents) to avoid unnecessary performance costs.